examples/iscsi/README.md

```diff
@@ -1,15 +1,24 @@
-# How to Use it?
-Here is my setup to setup Kubernetes with iSCSI persistent storage. I use Fedora 21 on Kubernetes node. 
+## Step 1. Setting up iSCSI target and iSCSI initiator
+**Setup A.** Setup on Fedora 21
 
-Install iSCSI initiator on the node:
+If you use Fedora 21 on Kubernetes node, then install iSCSI initiator on the node:
 
     # yum -y install iscsi-initiator-utils
    
    
 then edit */etc/iscsi/initiatorname.iscsi* and */etc/iscsi/iscsid.conf* to match your iSCSI target configuration.
 
-I mostly follow these [instructions](http://www.server-world.info/en/note?os=Fedora_21&p=iscsi&f=2) to setup iSCSI initiator and these [instructions](http://www.server-world.info/en/note?os=Fedora_21&p=iscsi) to setup iSCSI target.
+I mostly followed these [instructions](http://www.server-world.info/en/note?os=Fedora_21&p=iscsi&f=2) to setup iSCSI initiator and these [instructions](http://www.server-world.info/en/note?os=Fedora_21&p=iscsi) to setup iSCSI target.
 
+**Setup B.** Setup on Ubuntu 12.04 and Debian 7 on GCE
+
+GCE does not provide preconfigured Fedora 21 image, so I set up the iSCSI target on a preconfigured Ubuntu 12.04 image, mostly following these [instructions](http://www.server-world.info/en/note?os=Ubuntu_12.04&p=iscsi). I made necessary changes such as creating two logical volumes and two LUN, because the *iscsi.json* in this example expects such a setup.
+
+I had kernel mismatch problems when setting up iSCSI target on other preconfigured GCE images. Using the Ubuntu 12.04 image makes the life much easier.
```

Comment:
I moved the story to a readme in my repository and linked to it. Could you take another look? Thanks.